examples/react_2d_mp/react/src/components/MYTYSDKView.tsx

```diff
@@ -0,0 +1,49 @@
+import { useContext, useRef, useEffect } from "react"
+import { MYTYSDKContext, mytySDKContext } from "../context/MYTYSDKContext"
+import { Unity } from "react-unity-webgl"
+import { VideoDeviceContext, videoDeviceContext } from "../context/VideoDeviceContext";
+import useHolistic from "../hooks/useHolistic";
+
+const MYTYSDKView = () => {
+  const {unityContext} = useContext(mytySDKContext) as MYTYSDKContext;
+  const { setCamera } = useContext(videoDeviceContext) as VideoDeviceContext;
+  const cameraRef = useRef<HTMLVideoElement>(null);
+  const {startLoading} = useHolistic();
+
+  useEffect(() => {
+    if(cameraRef.current) {
+      setCamera(cameraRef.current);
+    }
+  }, [cameraRef, setCamera])
+
+  return <div style={{ position: 'relative', width: '720px', height: '480px' }}>
+        <Unity
+          style={{
```

Comment:
Add devicePixelRatio=2 to props of Unity. The default value is 1 and it causes poor quality of the rendered result.

Reply:
Thank you! I'll apply it right away